cmd/azmount/filemanager/azure.go

```diff
@@ -86,58 +87,59 @@ func AzureSetup(urlString string, urlPrivate bool, identity common.Identity) err
 	logrus.Infof("Connecting to Azure...")
 	u, err := url.Parse(urlString)
 	if err != nil {
-		return errors.Wrapf(err, "can't parse URL string")
+		return errors.Wrapf(err, "Can't parse URL string %s", urlString)
 	}
 
 	if urlPrivate {
 		// we use token credentials to access private azure blob storage the blob's
 		// url Host denotes the scope/audience for which we need to get a token
-		logrus.Infof("Using token credentials")
+		logrus.Info("Using token credentials to access private azure blob storage...")
 
 		var token common.TokenResponse
 		count := 0
+		logrus.Infof("Getting token for https://%s", u.Host)
 		for {
 			token, err = common.GetToken("https://"+u.Host, identity)
 
 			if err != nil {
-				logrus.Infof("can't obtain a token required for accessing private blobs. will retry in case the ACI identity sidecar is not running yet.")
+				logrus.Info("Can't obtain a token required for accessing private blobs. Will retry in case the ACI identity sidecar is not running yet...")
 				time.Sleep(3 * time.Second)
 				count++
 				if count == 20 {
-					return errors.Wrapf(err, "timeout of 60 seconds expired. could not obtained token")
+					return errors.Wrapf(err, "Timeout of 60 seconds expired. Could not obtain token")
 				}
 			} else {
-				logrus.Infof("token obtained: %s continuing", token.AccessToken)
+				logrus.Infof("Token obtained: %s \nContinuing...", token.AccessToken)
```

Comment:
Logs shouldn't span multiple lines like this.  Also, this access token is the same as a username + password.  Tracing logging only for the contents.